inst/tests/froll.Rraw

```diff
@@ -322,7 +321,7 @@ test(6000.0672, frollmean(c(1:2,NA,4:10), 4, hasNA=FALSE), c(rep(NA_real_, 6), 5
   "frollmean: processing algo 0 took.*",
   "frollfunR: processing.*took.*"
 ), warning="hasNA=FALSE used but NA.*are present in input, use default hasNA=NA to avoid this warning")
-test(6000.0673, frollmean(c(1:2,NA,4:10), 2, hasNA=FALSE), c(NA, 1.5, NA, NA, 4.5, 5.5, 6.5, 7.5, 8.5, 9.5), output=c(
+test(6000.0673, options = c(datatable.verbose=TRUE), frollmean(c(1:2,NA,4:10), 2, hasNA=FALSE), c(NA, 1.5, NA, NA, 4.5, 5.5, 6.5, 7.5, 8.5, 9.5), output=c(
```

Comment:
I think many of those tests testing the output will be changed in another PR, therefore this PR will introduces many non-auto mergable lines. Personally I would prefer, as agreed before 1.14.10, to focus on clearing out PR queue, and then in next release create new PRs. This is to reduce overall work required to deal with conflicts.

Reply:
How about excluding froll.Rraw then? tests.Rraw was excluded already for the same reason. Are any of the other scripts here involved in the major pending PRs?

A very conservative approach is to limit this PR to adding `options=` to `test()`, then do all conversions in follow-up(s).

Reply:
I don't recall any others but it is up to 3 years old so not difficult to miss some.

Your conservative approach makes sense. We could use this option when adding new tests, but then eventually, when PRs are cleared, deal with the historical tests.

Reply:
I am searching through all branches to find edits to each test file as well.

I think it is OK to ignore PRs in draft status, e.g. https://github.com/Rdatatable/data.table/pull/4517 which touches benchmark.Rraw.

Reply:
I'll write a follow-up to #5996 to make the changes in {benchmark,programming,types}.Rraw & leave {froll,nafill,other}.Rraw for 1.17.0.